pom.xml

```diff
@@ -1521,7 +1521,7 @@
             <configuration>
               <rules>
                 <enforceBytecodeVersion>
-                  <maxJdkVersion>1.7</maxJdkVersion>
+                  <maxJdkVersion>1.8</maxJdkVersion>
```

Comment:
note that HIFIO had the same problem, and was able to solve it using a per-module change - check out the pom in the jdk1.8-tests dir, it changes the jdk version